### ISSUE TITLE
val loss：nan, accuray：0

### ISSUE THREAD

**Issue by Yumisilence** [Holmeyoung/crnn-pytorch#72]
when I use my datasets and holmeyoung's code，I can train a model that works. But after I modified the structure of cnn，the loss was turn to nan. Why did it go wrong?

**Comment by la-la-la-1** [comment]
请问你解决了吗？
